PURL-SPECIFICATION.rst

```diff
@@ -427,11 +433,11 @@ Known ``qualifiers`` key/value pairs
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Note: Do not abuse ``qualifiers``: it can be tempting to use many qualifier
-keys but their usage should be limited to the bare minimum for proper package
+keys but their usage SHOULD be limited to the bare minimum for proper package
```

Comment:
This is an instruction to PURL spec authors. PURL implementations shouldn't be removing qualifiers to make the PURL more compact.